lib/datadog/tracing/contrib/grpc/formatting.rb

```diff
@@ -0,0 +1,115 @@
+# typed: ignore
```

Comment:
Minor: This comment was used by the Sorbet type checker, which we removed in #2647 (we only rely on Steep now, which does not need this comment).

TL;DR: You can remove this now :)

Reply:
Thanks for letting me know! I'll update the PR 👍